totalRP3/Modules/Dashboard/Dashboard.xml

```diff
@@ -192,32 +191,32 @@ https://raw.githubusercontent.com/Meorawr/wow-ui-schema/main/UI.xsd">
 					<OnClick function="HideParentPanel"/>
 				</Scripts>
 			</Button>
-			<EditBox parentKey="name" inherits="TRP3_TitledHelpEditBox">
+			<EditBox parentKey="Name" inherits="TRP3_TitledHelpEditBox">
 				<Size y="18"/>
 				<Anchors>
-					<Anchor point="TOP" relativePoint="BOTTOM" relativeKey="$parent.title" y="-10"/>
+					<Anchor point="TOP" relativePoint="BOTTOM" relativeKey="$parent.Title" y="-10"/>
 					<Anchor point="LEFT" relativePoint="LEFT" x="25"/>
 					<Anchor point="RIGHT" relativePoint="CENTER" relativeKey="$parent" x="-20"/>
 				</Anchors>
 			</EditBox>
-			<Frame parentKey="channelDropdown" inherits="TRP3_DropDownButtonTemplate" enableMouse="true" name="$parentChannelDropdown">
+			<Frame parentKey="ChannelDropdown" inherits="TRP3_DropDownButtonTemplate" enableMouse="true" name="$parentChannelDropdown">
```

Comment:
what happens if you delete the dropdown name

Reply:
It's safe to delete with the changes for new dropdowns.

```suggestion
			<Frame parentKey="ChannelDropdown" inherits="TRP3_DropDownButtonTemplate" enableMouse="true">
```

Reply:
Okay then.